src/fix_adapt.cpp

```diff
@@ -568,46 +573,49 @@ void FixAdapt::change_settings()
       // also scale rmass to new value
 
       if (ad->aparam == DIAMETER) {
-        int mflag = 0;
-        if (atom->rmass_flag) mflag = 1;
+				/* `mflag` unnecessary ? the test `if(!atom->radius_flag)` in `FixAdapt::init()` should perevent `atom->rmass_flag == false`. Unless there can be combinations of atom styles with `radius` but without `rmass`
```

Comment:
ok to delete mflag here (and remove these comments) - but please
add a test for rmass flag (similar to radius_flag_ in init()